qemu/tests/blockdev_mirror_hotunplug.py

```diff
@@ -1,15 +1,10 @@
 import time
 
 from virttest.qemu_monitor import QMPCmdError
-
 from provider.blockdev_mirror_nowait import BlockdevMirrorNowaitTest
 
 
 class BlockdevMirrorHotunplugTest(BlockdevMirrorNowaitTest):
-    """
-    Block mirror with hotunplug test
-    """
-
```

Comment:
The same comment as above, and it's better to make a brief introduction to the class